commerce-order-content-web/src/main/resources/META-INF/resources/pending_orders/edit_order.jsp

```diff
@@ -53,6 +53,80 @@ List<CommerceAddress> billingAddresses = commerceOrderContentDisplayContext.getB
 	<portlet:param name="mvcRenderCommandName" value="editCommerceOrder" />
 </portlet:actionURL>
 
+<commerce-ui:header
+	bean="<%= commerceOrderContentDisplayContext.getCommerceOrder() %>"
+	dropdownItems="<%= commerceOrderContentDisplayContext.getDropdownItems() %>"
+	headerActions="<%= commerceOrderContentDisplayContext.getHeaderActionModels() %>"
+	model="<%= CommerceOrder.class %>"
+	title='<%= LanguageUtil.format(request, "order-x", commerceOrderContentDisplayContext.getCommerceOrderId()) %>'
+/>
+
+<commerce-ui:panel
+	title="Panel title"
+>
+	<div class="row vertically-divided">
+		<div class="col-sm-4">
+			<commerce-ui:info-box
+				actionLabel="Edit"
+				actionTargetId="billing-address-modal"
+				elementClasses="py-3"
+				title="Billing Address"
+			>
+				PO Box 467 New York NY 10002
+			</commerce-ui:info-box>
+
+			<commerce-ui:info-box
+				actionLabel="Edit"
+				actionTargetId="shipping-address-modal"
+				elementClasses="py-3"
+				title="Shipping Address"
+			>
+				PO Box 467 New York NY 10002
+			</commerce-ui:info-box>
+		</div>
+
+		<div class="col-sm-4">
+			<commerce-ui:info-box
+				actionLabel="Edit"
+				actionTargetId="purchase-order-number-modal"
+				elementClasses="py-3"
+				title="Purchase Order Number"
+			>
+				#56735234
+			</commerce-ui:info-box>
+
+			<commerce-ui:info-box
+				actionLabel="Edit"
+				actionTargetId="request-delivery-date-modal"
+				elementClasses="py-3"
+				title="Request Delivery Date"
+			>
+				Aug 24, 2019
+			</commerce-ui:info-box>
+		</div>
+
+		<div class="col-sm-4">
+			<commerce-ui:info-box
+				actionLabel="Edit"
+				actionTargetId="edit-order-notes-modal"
+				elementClasses="py-3"
+				title="Order Date"
+			>
+				Jul 12, 2019
+			</commerce-ui:info-box>
+
+			<commerce-ui:info-box
+				actionLabel="Edit"
+				actionTargetId="edit-order-notes-modal"
+				elementClasses="py-3"
+				title="Order Notes"
+			>
+				Curabitur non nulla sit amet nisl tempus convallis quis ac lectus. Vestibulum ac diam sit amet quam vehicula elementum sed sit amet dui.
+			</commerce-ui:info-box>
+		</div>
+	</div>
+</commerce-ui:panel>
```

Comment:
I don't understand why this is here. Can you explain?

Reply:
Ethan! You're right, I've updated the pr removing that code